app/migrations/Version20151217104913.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Sylius\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20151217104913 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE sylius_product_attribute ADD storage_type VARCHAR(255) NOT NULL, ADD validation LONGTEXT NOT NULL COMMENT \'(DC2Type:array)\', CHANGE name code VARCHAR(255) NOT NULL');
```

Comment:
This comment applies to whole PR but we should not have a separate ``validation`` field. Validation configs should be kept in general attribute configuration because you may have options which affect both rendering and validation. And it also complicates the forms, no need for that separation. :)